src/claudia_launcher.py

```diff
@@ -832,10 +832,14 @@ def refreshAll(self):
 
                 last_pos += 1
 
-        if haveCarla and os.path.exists("/usr/" + self.lib_path + "/carla/libcarla_utils.so"):
-            utils = CarlaUtils("/usr/" + self.lib_path + "/carla/libcarla_utils.so")
+        if haveCarla and os.path.exists("/usr/{}/carla/libcarla_utils.so".format(self.lib_path)):
+            utils = CarlaUtils("/usr/{}/carla/libcarla_utils.so".format(self.lib_path))
             last_pos = 0
-            for i in range(utils.get_cached_plugin_count(PLUGIN_LV2, os.getenv("LV2_PATH", "~/.lv2:/usr/" + self.lib_path + "/lv2:/usr/local/" + self.lib_path + "/lv2"))):
+            for i in range(utils.get_cached_plugin_count(PLUGIN_LV2, 
+                os.getenv("LV2_PATH", 
```

Comment:
previous code had the default value stored in a variable.
if you do that, you should no longer need to split the string into multiple lines